types/yargs/index.d.ts

```diff
@@ -176,6 +176,7 @@ declare namespace yargs {
         ): Argv<T>;
         command(command: string | ReadonlyArray<string>, showInHelp: false, module: CommandModule<T, any>): Argv<T>;
         command(module: CommandModule<T, any>): Argv<T>;
+        command(modules: Array<CommandModule<any, any>>): Argv<T>;
```

Comment:
Let me know if the first type was `any` for a specific reason.

```suggestion
        command(modules: Array<CommandModule<T, any>>): Argv<T>;
```

Reply:
Thanks for the comment and I have made this change.

If you look at this commit, https://github.com/DefinitelyTyped/DefinitelyTyped/pull/64778/commits/2be2b8b845a049672597bd3376a88fd42deb83fc, I did have to remove the generic types from the test in order for it work. It seems like there's some issue with the `command` function overloading that's using the wrong type when I have the generic type defined there.